babel.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = (api) => {
+  // Cache configuration is a required option
+  api.cache(false);
+
+  const presets = [
+    '@babel/preset-typescript',
+    '@babel/preset-env',
+  ];
+
+  const plugins = [
+    ['add-module-exports', { addDefaultProperty: true }],
+    '@babel/plugin-transform-strict-mode',
+    '@babel/plugin-transform-runtime',
+    '@babel/plugin-proposal-json-strings',
```

Comment:
this is in preset-env according to https://babeljs.io/docs/en/babel-plugin-proposal-json-strings